portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +31,131 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Add a random fact about me to the page
+ */
+function addRandomFact() {
+    const facts = [
+        'I am a foodie',
+        'I have a collection of Starbucks mugs',
+        'I have a collection of "I LOVE" T-shirt from around the world',
+        'I have travelled to over 15 different countries'
+    ];
+
+    //Pick a random fact
+    const fact = facts[Math.floor(Math.random() * facts.length)];
+
+    //Add it to the page
+    const factContainer = document.getElementById('fact-container');
+    factContainer.innerText = fact;
+}
+
+isHobbiesDisplayed = false;
```

Comment:
Generally we try to avoid global variables as much as possible, it's very easy to accidentally create complicated and incorrect interactions between them (not that I think you have here, but it's just better style to avoid them).
What you could do instead of using these variables is put say 3 divs inside the about-me-container and turn their display on and off as necessary. It will have the same affect as the variables but it will be directly tied to the dom and make it much more obvious what's being controlled.

Reply:
I have fixed my code from the comments and have also changed the "display about me (hobbies, what I love, etc.)" function. I removed the global variables and, as I am planning to keep the list static, I moved the HTML code to the HTML file and only kept the show/hide feature to the JavaScript file. 

I also decided to combine it into one function, as it was otherwise quite repetitive, but let me know whether it is more confusing that way. 

(I also pushed a change I made yesterday but forgot to push. It's about the fun fact section, I just moved its location on the page).

Let me know whether there is anything else I could change/improve :)